api/src/main/java/org/pmiops/workbench/actionaudit/auditors/UserServiceAuditor.java

```diff
@@ -21,4 +22,7 @@ void fireAdministrativeBypassTime(
       Optional<Instant> newBypassTime);
 
   void fireAcknowledgeTermsOfService(DbUser targetUser, Integer termsOfServiceVersion);
+
+  void fireFreeTierDollarQuotaAction(
```

Comment:
nit: general convention would be something like `fireEditFreeTierQuota()`. Auditors always fire actions, so that part of the name doesn't narrow it down much.